subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoTaskExtension.java

```diff
@@ -88,6 +94,7 @@ public JacocoTaskExtension(Project project, JacocoAgentJar agent, JavaForkOption
     /**
      * Whether or not the task should generate execution data. Defaults to {@code true}.
      */
+    @Input
```

Comment:
I guess this doesn't have to be an input.

Reply:
I first didn't declare it as an input and then decided to declare it. I doesn't really matter.
Adding `@Input` seemed marginally better to me than adding `@Internal`.
WDYT?

Reply:
Yeah, I guess it's better to keep it as an `@Input` then.